b9lab-content/2-what-is-cosmos/02-blockchain-and-cosmos.md

```diff
@@ -0,0 +1,154 @@
+
+On a fateful October 31, 2008, an individual, or group, calling itself Satoshi Nakamoto proposed a **P2P network for a digital currency**, calling it **Bitcoin**. It introduced a novel consensus  mechanism called Proof-of-Work (PoW). Suddenly, it became possible to send online payments directly between parties **independently of financial institutions and trusted third parties**. Bitcoin became the first public, decentralized application.
+
+<ExpansionPanel title="I heard about PoW, tell me more">
+
+Proof-of-Work (PoW), used to achieve BFT, is best described as a cryptographic puzzle solved by a network's node, called miner. The puzzle is a task of pre-defined, arbitrary difficulty. At the scale of the network, the outcome is akin to a lottery with a single winning node.
+
+In most cases, the task consists of a search for an unknown, random number (in the jargon, a nonce). For it to be a winning nonce, when combined with ordered transactions in a block, the result ought to be a hash value matching a pre-defined criteria. Finding the nonce is evidence of considerable effort, or work, invested in the search. Each node uses its computing power to be the first to solve the puzzle, winning the right to author the latest block.
+
+Here, economic incentives come into play: The node that announces a solution first receives a reward. This incentivizes participation, investing computing power into solving the task, and helps maintain the network's functioning.
+
+</ExpansionPanel>
+
+With the introduction of Bitcoin, the **development of decentralized applications (DApps)** - applications built on a decentralized network - began. In the early days, developing DApps could only be done by **forking the Bitcoin codebase** or **building on it**. However, Bitcoin's monolithic codebase and limited, not user-friendly scripting language made DApp development a tedious and complex process for developers.
+
+<ExpansionPanel title="What's a Blockchain, really?">
+
+**Blockchain protocols** define programs that **hold a state** and that describe how to modify it according to the received inputs. Those inputs are called transactions.
+
+The consensus ensures that a blockchain has a **canonical transaction history**. It follows that the **state is deterministic**, meaning if you begin with the same genesis state and replicate all conducted changes, you always achieve the same state.
+
+Bitcoin's codebase may have been monolithic, but it nevertheless was a blockchain architecture, i.e. could be split into **three layers**:
+
+* **Network layer:** Tasked with discovering nodes and propagating transactions and consensus-related messages between single nodes;
+* **Consensus layer:** Runs the consensus protocol between the single nodes of a P2P network;
+* **Application layer:** In effect, running a state machine, which defines the application's state and updates it with the processing of transactions, in accordance with the result of the consensus.
```

Comment:
```suggestion
Although Bitcoin's codebase may have been monolithic, its blockchain architecture can be split into three layers:

* Network layer: Tasked with discovering nodes and propagating transactions and consensus-related messages between single nodes;
* Consensus layer: Runs the consensus protocol between the single nodes of a P2P network;
* Application layer: In effect, running a state machine that defines the application's state and updates it with the processing of transactions, in accordance with the result of the consensus.
```

Reply:
I wonder if we can talk about the use (overuse) of bolding? Our words are very strong and informative, so let's find a style that we all agree with and use consistently. Less is more.

Reply:
so Bitcoin architecture is no longer monolithic? (if it still is, we can use "codebase is monolithic")

Reply:
I'll have a look out at our overuse of bolding. We often like to use it to highlight the central theme of a paragraph.
I changed the part on the Bitcoin architecture.